src/main/paradox/docs/api/assets/realms/realm-added-ref.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": "https://bluebrain.github.io/nexus/contexts/realms.json",
+  "@id": "http://dev.nexus.ocp.bbp.epfl.ch/auth/realms/realm1",
```

Comment:
This id is weird. I think the id should be: https://nexus.example.com/v1/realms/realm1